udata/tests/test_discussions.py

```diff
@@ -358,6 +364,57 @@ def test_list_discussions_for(self):
 
         self.assertEqual(len(response.json["data"]), len(discussions))
 
+    def assertIdIn(self, json_data: dict, id_: str) -> None:
+        for item in json_data:
+            if item["id"] == id_:
+                return
+        self.fail(f"id {id_} not in {json_data}")
+
+    def test_list_discussions_org(self) -> None:
+        organization: Organization = OrganizationFactory()
+        user: User = UserFactory()
+        _discussion: Discussion = DiscussionFactory(user=user)
+        dataset = DatasetFactory(organization=organization)
+        dataservice = DataserviceFactory(organization=organization)
+        reuse = ReuseFactory(organization=organization)
+        discussion_for_dataset: Discussion = DiscussionFactory(subject=dataset, user=user)
+        discussion_for_dataservice: Discussion = DiscussionFactory(subject=dataservice, user=user)
+        discussion_for_reuse: Discussion = DiscussionFactory(subject=reuse, user=user)
+
+        response: TestResponse = self.get(url_for("api.discussions", org=organization.id))
+        self.assert200(response)
+        self.assertEqual(len(response.json["data"]), 3)
+        self.assertIdIn(response.json["data"], str(discussion_for_dataset.id))
+        self.assertIdIn(response.json["data"], str(discussion_for_dataservice.id))
+        self.assertIdIn(response.json["data"], str(discussion_for_reuse.id))
+
+    def test_list_discussions_sort(self) -> None:
+        user: User = UserFactory()
+        sorting_keys_dict: dict = {
```

Comment:
Was it too cumbersome to parametrize the tests?
It would additionally prevent cleaning the discussions manually

Reply:
It's not a matter of being cumbersome, but a matter of being in a file and class that doesn't use pytest, but rather unittest, and pytest's parametrization [isn't compatible](https://docs.pytest.org/en/latest/how-to/unittest.html#pytest-features-in-unittest-testcase-subclasses) :/

I didn't want to create another class that would test the same stuff, but that wouldn't inherit from unittest, as it seemed a bit clunky to me... What do you think?

Maybe I could rewrite the `DiscussionsTest` class to not use unittest, maybe in another PR?

Reply:
Ahah I think it is the kind of things I meant behind cumbersome.

Indeed, it would be the best to make these tests not use unittest in an upcoming PR :)
Maybe done class by class or by updating `TestCase` directly.